zerver/webhooks/opbeat/doc.md

```diff
@@ -0,0 +1,10 @@
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+Go to your organization settings page (`https://opbeat.com/<organization_name>/settings`),
+and copy your webhook URL to the **Hook target** under HTTP hook.
+
```

Comment:
Okay, so I would suggest including a couple of screenshots to ease the process and explaining how one might create the webhook from start to finish. Apart from that this looks good and simple enough! Thanks! :)